src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -610,9 +610,11 @@ public partial class TcpClient : System.IDisposable
         public System.Threading.Tasks.Task ConnectAsync(System.Net.IPAddress address, int port) { throw null; }
         public System.Threading.Tasks.Task ConnectAsync(System.Net.IPAddress[] addresses, int port) { throw null; }
         public System.Threading.Tasks.Task ConnectAsync(string host, int port) { throw null; }
+        public System.Threading.Tasks.Task ConnectAsync(System.Net.EndPoint remoteEP) { throw null; }
```

Comment:
This should be IPEndPoint, *not* EndPoint. Similarly below.

Reply:
Good catch.  "Tcp"Client, after all.

Reply:
Right.